packages/gatsby/src/utils/__tests__/__snapshots__/handle-flags.ts.snap

```diff
@@ -247,6 +252,9 @@ flags: {
 The following flags were automatically enabled on your site:
 - PARTIAL_RELEASE · (Umbrella Issue (test)) · test
 - PARTIAL_RELEASE_ONLY_NEW_LODASH · (Umbrella Issue (test)) · test
+
+There is one other flag available that you might be interested in:
+- YET_ANOTHER · (Umbrella Issue (test)) · test
```

Comment:
example of suggestion not being displayed due to bad condition check we had before